cypress/e2e/05_add_garden/add_garden_workflow.cy.js

```diff
@@ -0,0 +1,545 @@
+describe('Going through the Add Garden Workflow', function () {
+
+    beforeEach(() => {
+        cy.login();
+        cy.visit('http://localhost:8000/plugins/init-workflow');
+    });
+
+    it('Go through the workflow and populate all fields', function () {
+        cy.contains('Workflows');
+        cy.contains('Add Garden').click();
+        cy.wait(2000);
+
+        // Add Garden 1st page
+        cy.get('[aria-label="Start new or please select from below"]').click();
+        cy.wait(2000);
+        cy.get('.select2-results__option').contains('HA/01').click();
+        cy.get('.btn-success').contains('Start New').click();
+        cy.wait(2000);
+
+        // Start tab
+        cy.wait(4000);
+        cy.get('.card_component.resourceid').contains('HA Number');
+        cy.get('.form-control').should('be.disabled');
```

Comment:
This is the proper way to check if an element is disabled. It should be like this in the other tests as well